tests/tests.py

```diff
@@ -13,14 +17,24 @@
                      cached, cached_view, cached_as, cached_view_as
 from cacheops import invalidate_fragment
 from cacheops.templatetags.cacheops import register
+from cacheops.transaction import _function_queue, in_transaction
+
 decorator_tag = register.decorator_tag
 from .models import *
 
 
-class BaseTestCase(TestCase):
+# django.test.TestCase wraps individual tests in atomic blocks. Old tests were not written to check
+#  the change in functionality cacheop's transaction support implements when in atomic blocks.
+#  Changing to django.test.TransactionTestCase fixes this issue. Tests are not in atomic blocks.
+#  This does make tests run slower then before, since tables are truncated and fixtures reapplied
+#  between tests.
+class BaseTestCase(TransactionTestCase):
```

Comment:
Still unclear why all the tests should be wrapped into `TransactionTestCase`. This seems unnecessary.